docs/tutorial/README.md

```diff
@@ -0,0 +1,302 @@
+guestbook-v1-14                 2/2     Running   0          18h   172.20.1.19    192.168.1.108   <none>
+guestbook-v1-15                 2/2     Running   0          18h   172.20.0.13    192.168.1.106   <none>
+guestbook-v1-16                 2/2     Running   0          18h   172.20.0.150   192.168.1.107   <none>
+guestbook-v1-17                 2/2     Running   0          18h   172.20.0.151   192.168.1.107   <none>
+guestbook-v1-18                 2/2     Running   0          18h   172.20.0.14    192.168.1.106   <none>
+guestbook-v1-19                 2/2     Running   0          18h   172.20.1.20    192.168.1.108   <none>
+guestbook-v1-2                  2/2     Running   0          18h   172.20.1.15    192.168.1.108   <none>
+guestbook-v1-3                  2/2     Running   0          18h   172.20.0.147   192.168.1.107   <none>
+guestbook-v1-4                  2/2     Running   0          18h   172.20.0.146   192.168.1.107   <none>
+guestbook-v1-5                  2/2     Running   0          18h   172.20.1.16    192.168.1.108   <none>
+guestbook-v1-6                  2/2     Running   0          18h   172.20.0.148   192.168.1.107   <none>
+guestbook-v1-7                  2/2     Running   0          18h   172.20.0.9     192.168.1.106   <none>
+guestbook-v1-8                  2/2     Running   0          18h   172.20.0.10    192.168.1.106   <none>
+guestbook-v1-9                  2/2     Running   0          18h   172.20.0.11    192.168.1.106   <none>
+```
+
+Run this command to patch the statefulset to use the new image.
+
+`kubect apply -f https://raw.githubusercontent.com/kruiseio/kruise/master/docs/tutorial/v1/guestbook-statefulset-v2.yaml`
+
+In particular, the difference is that the image version is updated to `v2` and partition is set to `15`, meaning that the pods with 
+ordinal larger than or equal to `15` will be updated to v2. The rest pods will remain at `v1`
+```yaml
+spec:
+    ...
+      containers:
+      - name: guestbook
+-       image: openkruise/guestbook:v1
++       image: openkruise/guestbook:v2
+  podManagementPolicy: Parallel  # allow parallel updates, works together with maxUnavailable
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      # Do in-place update if possible, currently only image update is supported for in-place update
+      podUpdatePolicy: InPlaceIfPossible
+      # Allow parallel updates with max number of unavailable instances equals to 2
+      maxUnavailable: 3
++     partition: 15
+```
+
+Check the statefulset, find the statefulset has 5 pods updated
+```
+$ kubectl get sts.apps.kruise.io
+
+NAME           DESIRED   CURRENT   UPDATED   READY   AGE
+guestbook-v1   20        20        5         20      18h
+``` 
+
+Check the pods again. `guestbook-v1-15` to `guestbook-v1-19` are updated with `RESTARTS` showing `1` and IPs remain the same.
+
+```
+$ kubectl get pods -o wide | grep guestbook
```

Comment:
I suggest using  `kubectl get pod -L controller-revision-hash -o wide`.

With `controller-revision-hash`, users can see the reversion of Pod which clearly shows pod already been updated.